autonomy/cli/deploy.py

```diff
@@ -244,13 +245,6 @@ def run(build_dir: Path, no_recreate: bool, remove_orphans: bool) -> None:
 @deploy_group.command(name="from-token")
 @click.argument("token_id", type=int)
 @click.argument("keys_file", type=click.Path())
-@click.option("--rpc", "rpc_url", type=str, help="Custom RPC URL")
-@click.option(
-    "--sca",
-    "service_contract_address",
-    type=str,
-    help="Service contract address for custom RPC URL.",
-)
```

Comment:
explain plase

Reply:
Since we use the logic from `autonomy.chain` now this won't work with the new chain profile selection. I have already updated the docs for the same. To use custom RPC and contract address you will have use `--use-custom-chain`